zapcore/encoder.go

```diff
@@ -239,6 +239,8 @@ type EncoderConfig struct {
 	// Unlike the other primitive type encoders, EncodeName is optional. The
 	// zero value falls back to FullNameEncoder.
 	EncodeName NameEncoder `json:"nameEncoder" yaml:"nameEncoder"`
+	// Configures the field separator used by the console encoder.
```

Comment:
Let's mention the default.

```suggestion
	// Configures the field separator used by the console encoder. Defaults
	// to tab.
```